### PR TITLE
Update 01 29 2021 3

### PR DESCRIPTION
**Pull request by ilyalazarev**

### CODE REVIEW

File: src/assets/style/blockquote.scss [hunk 1]
```diff
@@ -1,7 +1,5 @@
 #app blockquote {
   margin: 0 0 2.0em;
   padding: 0 1.5em;
-  border-left: 0.25em solid var(--border-color);
-  transition: border-color .3s;
```
Comment (Selnapenek) [inline src/assets/style/blockquote.scss]: https://github.com/8base/Documentation/pull/48#discussion_r569719113

File: src/assets/style/scrollbar.scss [hunk 2]
```diff
@@ -4,12 +4,12 @@
...
-.sidebar::-webkit-scrollbar-thumb {
+/* .sidebar::-webkit-scrollbar-thumb {
   background: rgba(0,0,0,0);
   transition: background-color 2s;
 }
 
 .sidebar::-webkit-scrollbar-thumb:hover,
 .sidebar:hover::-webkit-scrollbar-thumb {
 	background:	var(--bg-teritary);
-}
```
Comment (Selnapenek) [inline src/assets/style/scrollbar.scss]: https://github.com/8base/Documentation/pull/48#discussion_r569721458

File: src/layouts/partials/LeftSidebar.vue [hunk 3]
```diff
@@ -17,6 +17,9 @@
   .sidebar-entry.menu-link_depth_4 {
     padding-left: 40px;
   }
+  .sidebar-entry-button {
+    cursor: pointer;
+  }
```
Comment (Selnapenek) [inline src/layouts/partials/LeftSidebar.vue:22]: you need to wrap the whole element, but it looks ok for now

**Comment by sebscholl** [comment]
I updated some of the code in here. Please make sure the expected behavior is still accurate and we can merge.

**Review by Selnapenek — APPROVED** [review]

**Pull request merged**